### DISCUSSION TITLE
Recordings from 2022 Workshop

### DISCUSSION THREAD

**Discussion by gwsteffens-az** [phoebe-project/phoebe2#931] [General]
Hi,
I've been watching the recordings from the 2022 workshop in order to get onboard with Phoebe 2.4. The recordings for Monday June 20th are great. However, they seem to be unavailable/missing from Monday the 21st onward. Would it be possible to make them available again? I can read the slides but the recordings add a lot to the experience.
Thanks!

**Comment by kecnry** [comment]
This was just fixed yesterday, but your browser may have an out-of-date cache.  Can you try accessing the page of a video that isn't working and doing a hard reload (ctrl+shift+r in most browsers) - does that work?  If not, can you post which link is still not working for you?

**Reply by kecnry** [reply]
Oh I see that the zoom links are broken (there only two seem to be zoom links though, the rest after the first day are videos hosted directly on the website).  @aprsa @kmhambleton - any idea if there are local copies of these we can host directly?

**Reply by gwsteffens-az** [reply]
After doing the browser hard reload, I'm experiencing the same issue from the 2nd day onward. A random sampling of the recordings on subsequent days all exhibit the same problem: "Sorry, this page couldn't be found". 
Thanks for looking into this.

**Reply by kecnry** [reply]
Have you tried reloading on the page that says "Sorry, this page couldn't be found"?

**Reply by gwsteffens-az** [reply]
Ah Ha! That worked with three exceptions.  Below is a screen capture with a red line through the recordings that are still not available for Part 1. I see two zoom related ones and another non-zoom one on June 23rd.  All the recordings are available for Part 2. 

![image](https://github.com/user-attachments/assets/ded21df6-e270-4d62-9ecb-c6d607cb6265)

Thanks again for your help.
